### ISSUE TITLE
Players unable to join

### ISSUE THREAD

**Issue by darkbro100** [dmulloy2/ProtocolLib#641]
After updating viaversion from version 2.1.0 to 2.1.1, players were unable to join our server and a large error was thrown in console everytime someone tried to connect no matter what version they tried. Tried upgrading protocollib from 4.1.0 to 4.4.0 to see if these issues still happened, unfortunately they did. We tried downgrading both viaversion and protocollib to their original versions to see if that would fix the issue but that also did not work, we also removed viaversion entirely to see if maybe viaversion was potentially causing an issue but even without viaversion players were still unable to join on 1.8. Haven't seen this issue anywhere else so I thought I'd post it here

Debug paste link:
https://pastebin.com/cYFhgwEW <--- /protocol dump
https://pastebin.com/MDQL67Aj <--- error thrown when players join

Description and relevant errors:
We do run a custom fork of spigot 1.8.8, but no changes have been made to it in over 4 months and we were running viaversion 2.1.0 and protocollib 4.1.0 with no issues.

**Comment by MiniDigger** [comment]
try reproducing it on spigot/paper. its nearly impossible to look into it if you modified the server.

also, a headsup, if you modify spigot/paper, please change both the server mod name (MinecraftServer getServerMod) and the implementation name (CraftServer getName) so that ppl know you are not using offical builds.

have you tried just running protocollib or just running via? also, report this to the via guys too (after you tried my suggestions), there seems to be going something wrong on their side too.

**Comment by darkbro100** [comment]
We haven't been able to reproduce this on a normal spigot 1.8.8 build / paper 1.13.2 build.

What makes this whole thing weird, however, is that nothing has really been changed...

Our custom fork hasn't had commit made to it for almost four months (Last commit was made 2-28-19), and we also don't really use it too frequently because we've been trying to move our servers off that fork because of how outdated it is, but we also can't just move entirely off it either.

We used to run ProtocolLib 3.6.5 and that ran fine for the longest time, then we updated to 4.1.0 with no problems when we started running LibsDisguise. After ViaVersion initially broke, we tried updating to 4.4.0 to see if that would fix the issues but nothing came about it. Eventually we realized this issue was completely independent of ViaVersion. I already submitted an issue to their git as well but got a similar response.
